spyder/plugins/completion/kite/client.py

```diff
@@ -68,6 +68,13 @@ def get_languages(self):
             response = ['python']
         return response
 
+    def get_onboarding_file(self):
+        verb, url = KITE_ENDPOINTS.ONBOARDING_ENDPOINT
+        success, response = self.perform_http_request(verb, url)
+        if response is None:
```

Comment:
I think that maybe this `if` isn't needed

Reply:
Definitely not needed!